File: Realt.Parser/Model/Property.cs
using System;
using Newtonsoft.Json;

namespace Realt.Parser.Model
{
    public class Property
    {
        public long Id { get; set; }

        public int? RoomTotal { get; set; }
        public int? RoomSeparate { get; set; }
        public bool Shared { get; set; }
        public int? Year { get; set; }
        public double SquareTotal { get; set; }
        public double? SquareLiving { get; set; }
        public double? SquareKitchen { get; set; }
        public int? Floor { get; set; }
        public int? FloorTotal { get; set; }
        public string Type { get; set; }
        public string Balcony { get; set; }
        public string District { get; set; }
        public string Address { get; set; }
        public int? PriceUsd { get; set; }
        public int? PriceByn { get; set; }
        public DateTime Created { get; set; }

        public string Error { get; set; }

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}

File: realt-stats-api/src/main/java/com/realt/statsapi/controllers/dto/Property.java
package com.realt.statsapi.controllers.dto;

import lombok.Getter;
import lombok.Setter;
import org.springframework.format.annotation.DateTimeFormat;

import java.time.LocalDate;

public class Property {
    @Getter
    @Setter
    private long id;
    @Getter
    @Setter
    private Integer roomTotal;
    @Getter
    @Setter
    private Integer roomSeparate;
//    @Getter
//    @Setter
//    private Boolean shared;
    @Getter
    @Setter
    private Integer year;
    @Getter
    @Setter
    private Integer yearFrom;
    @Getter
    @Setter
    private Integer yearTo;
//    @Getter
//    @Setter
//    private Boolean isNew;
    @Getter
    @Setter
    private Double squareTotal;
    @Getter
    @Setter
    private Double squareLiving;
    @Getter
    @Setter
    private Double squareKitchen;
    @Getter
    @Setter
    private Integer floor;
    @Getter
    @Setter
    private Integer floorTotal;
    @Getter
    @Setter
    private String type;
    @Getter
    @Setter
    private String balcony;
    @Getter
    @Setter
    private String district;
    @Getter
    @Setter
    private String address;
    @Getter
    @Setter
    private Integer priceUsd;
    @Getter
    @Setter
    private Integer priceByn;
    @Getter
    @Setter
    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
    private LocalDate created;
}
